static/app/data/platformCategories.tsx

```diff
@@ -194,6 +194,7 @@ export const withPerformanceOnboarding: Set<PlatformKey> = new Set([
 export const withoutPerformanceSupport: Set<PlatformKey> = new Set([
   'elixir',
   'minidump',
+  'nintendo',
```

Comment:
@ArthurKnaus although I added Nintendo here I see performance as action item:
![Screenshot 2024-04-03 at 2 37 12 PM](https://github.com/getsentry/sentry/assets/132939361/aca72d26-aef8-404e-8529-f3b16b3f9cce)

I also see performance tab with no "not supported" banner. 
![Screenshot 2024-04-03 at 2 40 02 PM](https://github.com/getsentry/sentry/assets/132939361/7b4148ad-5af7-4445-a360-3c8096464484)
![Screenshot 2024-04-03 at 2 40 23 PM](https://github.com/getsentry/sentry/assets/132939361/a8c0b560-e5e3-4576-a023-685d1dfda711)

Looked into code and doesn't seem like withoutPerformanceSupport is actually used that widely. Can I go ahead and add banners like this to performance tabs?
![Screenshot 2024-03-27 at 1 16 07 PM](https://github.com/getsentry/sentry/assets/132939361/a0aafb72-d053-40af-a78e-39f1a68909e3)

Reply:
Good thinking! 👍
Though, I would merge this PR and  create a separate one that someone from performance reviews.